index.ipynb

```diff
@@ -492,42 +403,70 @@
    "outputs": [],
    "source": [
     "# Filter the height list using the above function\n",
-    "filtered_height = None"
+    "# Replace None with appropriate code\n",
+    "filtered_height = None\n",
+    "\n",
+    "len(filtered_height) # 4347"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": 9,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "4347"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
     "# __SOLUTION__ \n",
-    "# Filter the height list using above function\n",
-    "filtered_height = filter_list(height)"
+    "filtered_height = filter_height_outliers(height)\n",
+    "\n",
+    "len(filtered_height)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now that we have filtered the outliers out of our data, let's recreate our histogram with 8 bins using our filtered data. \n",
+    "Now that we have filtered the outliers out of our data and reduced the size of the dataset from 4785 to 4347, let's recreate our histogram with 8 bins using our filtered data. \n",
     "\n",
-    "**_NOTE_**: You do not need to reimport matplotlib -- once it's been imported, it's stored in memory and can be accessed whenever we like in other cells. "
+    "**_NOTE_**: You do not need to reimport `matplotlib` -- once it's been imported, it's stored in memory and can be accessed whenever we like in other cells. "
```

Comment:
Since you capitalized it in an earlier cell, `Matplotlib` here to be consistent

Reply:
The convention is
> Matplotlib 
when referring to the library (i.e. project), and
> `matplotlib`
when referring to the module that is actually imported.

In the added commit I clarified that specifically we do not need to reimport `plt` (i.e. an alias for a submodule, so it's surrounded by backticks but not capitalized)

Reply:
Thanks for clarifying that convention